.clang-format

```diff
@@ -0,0 +1,8 @@
+BasedOnStyle: LLVM
+AlignAfterOpenBracket: AlwaysBreak
+AlignEscapedNewlines: DontAlign
+AllowAllParametersOfDeclarationOnNextLine: false
+PenaltyReturnTypeOnItsOwnLine: 300
+BinPackParameters: false
+ColumnLimit: 100
```

Comment:
Did we decide to go with 120 on Friday, or keeping it 100? Just checking.

Reply:
Default value is 80. IMHO it's too small. I would prefer 100 so we can compare code side-by-side. IMHO, 120 is too long but I'm ok with 120 as well.